### PR TITLE
docker 1.11.2

### PR DESCRIPTION
**Pull request by ryansch**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-versions/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?
- [x] Does your submission adhere to Versions "Acceptable Formulae" [guidelines](https://github.com/Homebrew/homebrew-versions/blob/07b17dfb9f1/README.md#acceptable-formulae)?

-----

Docker 1.12 is in RC and I need a way to pin my local install to 1.11.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Please delete `docker110.rb` as part of this PR, thanks!

File: docker111.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class Docker111 < Formula
+  desc "Pack, ship and run any application as a lightweight container"
+  homepage "https://www.docker.com/"
+  url "https://github.com/docker/docker.git",
+      :tag => "v1.11.2",
+      :revision => "b9f10c951893f9a00865890a5232e85d770c1087"
+
+  head "https://github.com/docker/docker.git"
```
Comment (DomT4) [inline docker111.rb]: Unless you're planning to peg it to the 1.11.x branch Versions formulae don't have generic `head`s.

Comment (ryansch) [inline docker111.rb]: Good catch.

**Comment by ryansch** [comment]
@MikeMcQuaid I'm not a big fan of that idea.  I'd rather keep 1.10 and 1.11 at least until 1.12 is final.  A lot of the ecosystem is only now rolling out 1.11 support which means some people are still going to need 1.10 (this was our case until last week).

**Comment by MikeMcQuaid** [comment]
Sorry, you can delete `docker171.rb`; the versioning had me confused as to which was newer.

**Comment by apjanke** [comment]
#1328 added `docker111` and superseded this PR. Thank you for the contribution regardless!

**Pull request closed without merging**